sdk/test/Services/CloudFront/UnitTests/Custom/CookieSignerTest.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
```

Comment:
There were no existing tests for `AmazonCloudFrontCookieSigner`. 

I added these to make sure I didn't break anything when removing the duplicated call (they passed before and after making the change).